### PR TITLE
CSP-1227 Implements new blob triggered function for QAR Import

### PR DESCRIPTION
**Pull request by DevenKShah**

### CODE REVIEW

File: src/SFA.DAS.Roatp.Jobs.UnitTests/Extensions/NationalAchievementRatesExtensionsTests/ToAgeEnumTests.cs [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+    [TestCase("", Age.Unknown)]
+    [TestCase("  ", Age.Unknown)]
+    [TestCase("rubbish", Age.Unknown)]
+    [TestCase("16-18", Age.SixteenToEighteen)]
+    [TestCase(" 16-18 ", Age.SixteenToEighteen)]
+    [TestCase("19-23", Age.NineteenToTwentyThree)]
+    [TestCase("24+", Age.TwentyFourPlus)]
+    [TestCase("total", Age.AllAges)]
+    [TestCase("TOTAL", Age.AllAges)]
+    public void ThenConvertsStringToAgeEnum(string value, Age expected)
```
Comment (ManishAgarwalDfE) [inline src/SFA.DAS.Roatp.Jobs.UnitTests/Extensions/NationalAchievementRatesExtensionsTests/ToAgeEnumTests.cs:20]: Is there no "when" condition before the test method?

File: src/SFA.DAS.Roatp.Jobs.UnitTests/Extensions/NationalAchievementRatesExtensionsTests/ToApprenticeshipLevelEnumTests.cs [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    [TestCase("", ApprenticeshipLevel.Unknown)]
+    [TestCase(" ", ApprenticeshipLevel.Unknown)]
+    [TestCase("rubbish", ApprenticeshipLevel.Unknown)]
+    [TestCase("advanced level", ApprenticeshipLevel.Three)]
+    [TestCase("Advanced Level", ApprenticeshipLevel.Three)]
+    [TestCase(" advanced level ", ApprenticeshipLevel.Three)]
+    [TestCase("higher level", ApprenticeshipLevel.FourPlus)]
+    [TestCase("intermediate level", ApprenticeshipLevel.Two)]
+    [TestCase("total", ApprenticeshipLevel.AllLevels)]
+    public void ThenConvertsStringToApprenticeshipLevelEnum(string value, ApprenticeshipLevel expected)
```
Comment (ManishAgarwalDfE) [inline src/SFA.DAS.Roatp.Jobs.UnitTests/Extensions/NationalAchievementRatesExtensionsTests/ToApprenticeshipLevelEnumTests.cs:20]: same comment as above

**Review by ManishAgarwalDfE — APPROVED** [review]

File: src/SFA.DAS.Roatp.Jobs.UnitTests/Extensions/NationalAchievementRatesExtensionsTests/ToAgeEnumTests.cs (hunk 1 above)
Comment (DevenKShah) [reply src/SFA.DAS.Roatp.Jobs.UnitTests/Extensions/NationalAchievementRatesExtensionsTests/ToAgeEnumTests.cs]: fixed

File: src/SFA.DAS.Roatp.Jobs.UnitTests/Extensions/NationalAchievementRatesExtensionsTests/ToApprenticeshipLevelEnumTests.cs (hunk 2 above)
Comment (DevenKShah) [reply src/SFA.DAS.Roatp.Jobs.UnitTests/Extensions/NationalAchievementRatesExtensionsTests/ToApprenticeshipLevelEnumTests.cs]: fixed

**Comment by DevenKShah** [comment]
This will be merged with #150

**Pull request closed without merging**